### PR TITLE
Detect credential access for macOS via Keychain

### PR DESCRIPTION
**Pull request by tim1234567**

### CODE REVIEW

File: rules/linux/macos_creds_from_keychain.yml [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+author: Tim Ismilyaev, oscd.community
+date: 2020/10/19
+references:
+    - https://github.com/redcanaryco/atomic-red-team/blob/master/atomics/T1555.001/T1555.001.md
+logsource:
+    category: process_creation
+    product: macos
+detection:
+    any_keychain_call:
+        ProcessName|startswith: 'security'
```
Comment (yugoslavskiy) [inline rules/linux/macos_creds_from_keychain.yml:14]: Hello @tim1234567! Thank you for your contribution!
Could you please change 

```
        ProcessName|startswith: 'security'
```

to the

```
        ProcessName|endswith: '/security'
```

as the full path to the binary (which is going to be logged) is `/usr/bin/security`? 

Thanks!

Comment (e6e6e) [reply rules/linux/macos_creds_from_keychain.yml:14]: Hi @tim1234567!
Could you make the changes that were recommended to you above?
Thanks!

Comment (tim1234567) [reply rules/linux/macos_creds_from_keychain.yml:14]: Thanks, Dan and @e6e6e, for the feedback. I'll push changes in the next few days. Sorry for the delay.

Comment (yugoslavskiy) [reply rules/linux/macos_creds_from_keychain.yml]: thanks!

**Comment by yugoslavskiy** [comment]
Thank you very much, @tim1234567! Well done!

**Review by yugoslavskiy — APPROVED** [review]
@thomaspatzke @Neo23x0 guys, I believe that this PR could be merged (:
please let us know what do you think about it if there is anything you would like to change.

**Comment by Neo23x0** [comment]
We don't use the field `ProcessName`. I'll change that. 

Here is a list of valid field names: 
https://github.com/Neo23x0/sigma/wiki/Taxonomy

**Comment by Neo23x0** [comment]
I've also change the rule to include sub commands listed in the man page for `security`. 
The rule gets much tighter and better that way, since `security` can be used for all kinds of legitimate and less suspicious tasks. 

https://gist.github.com/Capybara/6228955

**Comment by yugoslavskiy** [comment]
Hello @Neo23x0!

> We don't use the field ProcessName. I'll change that.

Sorry, I didn't notice that you've restructured the rules to unify field names. 
Let me fix this specific issue all at once after we merge all the PRs to the `oscd` branch.  

> I've also change the rule to include sub commands listed in the man page for security.

Thanks!

**Pull request merged**